src/raftstore/store/worker/split_check.rs

```diff
@@ -24,31 +24,50 @@ use raftstore::Result;
 use raftstore::coprocessor::CoprocessorHost;
 use raftstore::store::engine::{IterOption, Iterable};
 use raftstore::store::{keys, Callback, Msg};
-use storage::{CfName, LARGE_CFS};
+use storage::{CfName, CF_WRITE, LARGE_CFS};
 use util::escape;
 use util::transport::{RetryableSendCh, Sender};
 use util::worker::Runnable;
 
 use super::metrics::*;
 
 #[derive(PartialEq, Eq)]
-struct KeyEntry {
+pub struct KeyEntry {
     key: Option<Vec<u8>>,
     pos: usize,
     value_size: usize,
+    from_write_cf: bool,
```

Comment:
I think you can just put the `cf_name` here.

Reply:
I think here `bool` is more friendly than `Vec<u8>`  in performance?

Reply:
How about `CfName`?